### PR TITLE
WIP Add runtime estimates

### PR DESCRIPTION
**Pull request by qubist**
Same changes as in 0b63bbe and 42121a9, just a different quote standard 👍

### CODE REVIEW

File: slowmovie.py [hunk 1]
```diff
@@ -90,10 +90,42 @@ def video_info(file):
...
+
+# Calculate how long it'll take to play a video.
+# output value: "d[ay]", "h[our]", "m[inute]", "s[econd]", "all"; omit for an automatic guess
+def estimate_runtime(delay, increment, videoLengthS, videoFPS, output="guess"):
+    # Recurse to generate all estimates in one string
+    if output == "all":
+        return f"{estimate_runtime(delay, increment, videoLengthS, videoFPS, 's')} / {estimate_runtime(delay, increment, videoLengthS, videoFPS, 'm')} / {estimate_runtime(delay, increment, videoLengthS, videoFPS, 'h')} / {estimate_runtime(delay, increment, videoLengthS, videoFPS, 'd')}"
+
+    # Calculate runtime lengths in different units
+    frames = videoLengthS*videoFPS
```
Comment (missionfloyd) [inline slowmovie.py:106]: Why not pass this `videoInfo['frame_count']` instead?

Comment (qubist) [reply slowmovie.py]: I'm not sure 😅  That's way simpler!

Comment (qubist) [reply slowmovie.py]: Changed: a2baaf5

**Review by robweber — APPROVED** [review]
Looks good - I like the "all" option.

**Pull request merged**